lab-trent/README.md

```diff
@@ -0,0 +1,10 @@
+# About

```

Comment:
Nice work on the doc here! How about a little more detail on things like commands to run for connecting one or more clients?